src/Sentry/Platforms/Android/NativeOptions.cs

```diff
@@ -272,6 +272,30 @@ public class NativeSentryReplayOptions
             public double? SessionSampleRate { get; set; }
             public bool MaskAllImages { get; set; } = true;
             public bool MaskAllText { get; set; } = true;
+
+            /// <summary>
+            /// <para>
+            /// Apps with complex user interfaces, consisting of hundreds of visual controls on a single page, may experience
+            /// performance issues due to the overhead of detecting custom masking of visual elements for Session Replays.
+            /// </para>
+            /// <para>
+            /// In such cases you have a few different options:
+            /// <list type="bullet">
+            ///   <item>
+            ///     <description>Disable Session Replays entirely</description>
+            ///   </item>
+            ///   <item>
+            ///     <description>Mask all text and all images</description>
+            ///   </item>
+            ///   <item>
+            ///     <description>Disable custom session replay masks (so nothing gets masked)</description>
+            ///   </item>
+            /// </list>
+            /// Set this option to <c>true</c> to disable custom session replay masks.
```

Comment:
Why would someone have custom masking (which requires extra code) and then flip this flag?

Reply:
We decided to make it an opt-in instead, to not pay the price for project that don't actively use custom masks. So for a new project / projects just starting to use custom masks, the feature must me activated alongside.

Existing applications already using custom masks now need to additionally also opt-in, but the feature is experimental so we're fine with the behavioral change.